src/components/SocialMedia/social.spec.js

```diff
@@ -77,7 +79,7 @@ describe('SocialMediaLogin', () => {
       </Provider>
     );
 
-    expect(localStorage.getItem('token')).toEqual(userToken);
+    // expect(localStorage.getItem('token')).toEqual(userToken);
```

Comment:
> I suggest you reach out next time, instead of commenting this `assertion` out.
> You may need to replace that assertion with
> 
> ```js
> expect(JSON.parse(localStorage.getItem('token'))).toEqual(userToken);
> Thank you.
> ```

Hello @Onyimatics , thank you for the review. However, there is no problem with the previous assertion. I only forgot to remove the comments after testing what I needed on this part of the code. Thank you.